### PR TITLE
add links to services in local provider

### PR DESCRIPTION
**Pull request by slaskis**
It adds a link like `OTHER_URL=http://other.test-links.convox` and I'm not sure if this is the correct urls but http seems to work better even with the convox ca installed.

I got my test in #2853 to now show `OTHER_URL` when running this local rack.

### CODE REVIEW

File: provider/local/converge.go [hunk 1]
```diff
@@ -361,6 +361,10 @@ func (p *Provider) resourceContainers(resources manifest.Resources, app, release
 	return cs, nil
 }
 
+func (p *Provider) serviceURL(app, name string) (string, error) {
+	return fmt.Sprintf("http://%s.%s.%s", name, app, p.Rack), nil
```
Comment (ddollar) [inline provider/local/converge.go]: This should probably be `https://`

File: provider/local/converge.go [hunk 2]
```diff
@@ -388,6 +392,20 @@ func (p *Provider) serviceContainers(services manifest.Services, app, release st
...
+		// add links
+		for _, sl := range s.Links {
+			for _, s := range m.Services {
+				if s.Name == sl {
+					u, err := p.serviceURL(app, s.Name)
+					if err != nil {
+						return nil, err
+					}
+
+					e[fmt.Sprintf("%s_URL", strings.ToUpper(sl))] = u
```
Comment (ddollar) [inline provider/local/converge.go]: Seems like this will break on services with dashes in the name, e.g. `my-service`

Comment (slaskis) [reply provider/local/converge.go]: oh yeah. good call!

Comment (slaskis) [reply provider/local/converge.go]: and it seems to be on par with the aws provider 

https://github.com/convox/rack/blob/master/provider/aws/formation/app.json.tmpl#L421

https://github.com/convox/rack/blob/master/provider/aws/template.go#L103

perhaps they should change to use upper / upperName instead?

Comment (slaskis) [reply provider/local/converge.go]: or no nvm...that seems to be "the other way around"

Comment (ddollar) [reply provider/local/converge.go]: Looks like you're right, go ahead and ignore this for now, will have to fix globally

**Pull request closed without merging**